frog/imports/ui/GraphEditor/utils/export.js

```diff
@@ -34,10 +35,11 @@ const doImportGraph = graphStr => {
   const graphObj = JSON.parse(graphStr.target.result);
   const importNo = getGlobalSetting('importNo') || 0;
   setGlobalSetting('importNo', importNo + 1);
-  const graphId = addGraph(
-    graphObj.graph.name + ' ' + importNo,
-    graphObj.graph
-  );
+  const graphId = addGraph(graphObj.graph.name + ' ' + importNo, {
+    ...graphObj.graph,
+    _id: uuid(),
```

Comment:
given the function `api/graphs/addGraph`:
```
export const addGraph = (
  name: string = 'undefined',
  graph: ?Object
): string => {
  const id = uuid();
  Graphs.insert({ ...graph, _id: id, name, createdAt: new Date() });
  return id;
};
```

it seems unnecessary to pass along an `_id`, but I understand the concern with passing a null sessionId